app/assets/stylesheets/components/header.scss

```diff
@@ -100,11 +100,13 @@
 
   &__trigger {
     all: unset;
-    padding: var(--su-1);
+    padding: var(--su-0);
```

Comment:
4px seemed like a lot of padding inside a blue focus outline, but that was the smallest variable listed. Happy to rename or rethink.

Reply:
That extra variables returns 2.4px which looks a bit off (since most screens don't really respect fractions):

![image](https://user-images.githubusercontent.com/108287/100877519-30eeb600-34a9-11eb-8dbd-64820a6a21ed.png)

You can easily notice more padding on the right comparing to the left one. So in this case I'd rather just use one-off like:

```
padding: 2px;
or
padding: calc(var(--su-1) / 2);
```